package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "web-vitals",
-  "version": "5.0.0-rc.0",
+  "version": "5.0.0-rc.1",
```

Comment:
This will get automatically updated by the release script.

Reply:
Oh I knew that worked for major/minor/patch but wasn't sure that also worked for RC patches.

Reply:
Reverted